src/stable.rs

```diff
@@ -83,32 +85,32 @@ impl fmt::Display for TokenStream {
                 write!(f, " ")?;
             }
             joint = false;
-            match tt.kind {
-                TokenNode::Group(delim, ref stream) => {
-                    let (start, end) = match delim {
+            match *tt {
+                TokenTree::Group(ref g) => {
```

Comment:
I would prefer to use the Syn match naming convention here, https://docs.rs/syn/0.12.15/syn/enum.Expr.html#syntax-tree-enums. For example `TokenTree::Op(ref tt) => ... tt.op() ...` to avoid `op.op()`.